content/en/Platform Deep Dive/Cobalt Account/account-recovery.md

```diff
@@ -81,18 +65,6 @@ One-time codes that your authenticator app generates are time-based. If the code
 - Enter the code promptly, within 30 seconds as it appears in the authenticator app.
 - Make sure the time is synchronized between the devices. If you manually changed the time zone on your mobile device earlier, select the system time zone.
 
-### "Remember This Device" Doesn't Work
```

Comment:
this is still there with 2FA, https://www.loom.com/i/fdfac54b4367479e9757a1040c0942b7. 

Remember me for 1 week is NO LONGER on the SIgn in Page here https://www.loom.com/i/88c59a93b3b2434fafc85cdcd85c89bb

Reply:
Added this back.